contracts/Staking.sol

```diff
@@ -482,6 +484,18 @@ contract Staking is IStaking, Initializable, OwnableUpgradeable, Constants {
         }
     }
 
+    function slashIndexer(address _indexer, uint256 _amount) external {
+        require(msg.sender == settings.getDisputeManager(), 'Only DisputeManager');
+        require(_amount <= getAfterDelegationAmount(_indexer, _indexer));
+
+        delegation[_indexer][_indexer].valueAt -= _amount;
+        totalStakingAmount[_indexer].valueAt -= _amount;
+        delegation[_indexer][_indexer].valueAfter -= _amount;
+        totalStakingAmount[_indexer].valueAfter -= _amount;
+
+        IERC20(settings.getSQToken()).safeTransfer(msg.sender, _amount);
```

Comment:
We may need a slashDestination config in staking contract other than send to dispute contract directly